app/assets/v2/css/profile.css

```diff
@@ -306,3 +310,84 @@
 .no-profile-textcolor {
   color: #666666;
 }
+
+#float_avatar_image{
+  z-index: 9999999;
```

Comment:
can we put just 9 or something lower so we don't if modal or popovers comes in some point don't get behind?